index.html

```diff
@@ -0,0 +1,114 @@
+
+    <!-- Google Font Montserrat Prompt -->
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:300Prompt&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="main.css">
+    <title>Html Forms</title>
+</head>
+
+<!-- cp am 1 -->
+
+<body>
+    <header>
+        <nav class="signin-section">
+            <div class="signin-info">
+                <p>Already have an Intuit account?</p>
+                <p>I forgot my User ID or Password</p>
+            </div>
+            <div class="sign-btn">
+                <button type="submit" class="sign-in-btn"><span class="sign-in-lock-icon"></span>
+
+                    Sign in </button>
+            </div>
+        </nav>
+    </header>
+
+    <!-- cp ak 2 -->
+
+    <!-- Logo Section -->
+    <div class="logos-section">
+        <div class="logo"></div>
+    </div>
+
+    <!-- Three Icon section -->
+    <div class="main-content">
+        <div class="logo-icons-section">
+            <img src="assets/intuit_3_logo.png" alt="">
+        </div>
+    </div>
+
+    <!-- Form Section -->
+    <main class="form-section">
+        <div class="form-body">
+            <form action="#" method="post">
+                <div class="form-header">
+                    <h2 class="sub-title">Create an Intuit account</h2>
+                    <p>One account for everything Intuit, including Mint.</p>
+                    <p class="learn-more">Learn more</p>
+                </div>
+                <div class="form-input">
+                    <label for="email">Email address</label>
+                    <input id="email" type="email" name="email">
+                    <div></div>
+
+                    <label for="phone">Phone (recommended)</label>
+                    <input id="phone" type="tel" name="phone">
+                    <p>Standard call, messaging or data rates may apply</p>
+                    <div></div>
+
+                    <label for="password">Password</label>
+                    <input id="password" type="password" name="password">
+                    <!-- <img src="assets/lock-left-grey.png"
+                        alt="open lock"> -->
+
+                    <!-- <span class="lock"></span> -->
+                    <div></div>
+
+                    <button type="submit"><span class="sign-in-lock-icon"></span> Create Account</button>
+                    <div></div>
+
+                    <cite>By clicking Create account, you agree to our</cite>
+                    <cite> <span>Terms</span> and have read and acknowledge our <span>Privacy</span> </cite>
+                    <cite> <span>Statement.</span></cite>
+                    <div></div>
+
+                    <div class="form-footer">
+                        <p>Invisible reCAPTCHA by Google <span>Privacy Policy</span> and <span>Terms of Use.</span></p>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </main>
+
+    <!-- cp am 2 -->
+

```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history